.github/workflows/typing.yml

```diff
@@ -22,10 +22,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install mypy==0.812
+        python -m pip install mypy
```

Comment:
It is a bit odd to add the type stubs to `dev-requirements`, but not `mypy` 🤔 Should we either add `mypy` as well, or create separate `type-requirements`?

Reply:
Yes I agree, lets make a type-requirements file :+1: